README.md

```diff
@@ -1,7 +1,15 @@
-# notation-hashicorp-vault
-Hashicorp Vault signing plugin for [Notation](https://notaryproject.dev/)
+# HashiCorp Vault plugin for Notation
+
+This repository contains the implementation of the HashiCorp Vault signing plugin for [Notation](https://notaryproject.dev/). This project is still in early development status.
```

Comment:
Suggest using link of `notation` repo

Reply:
I will revise `Notation` to `Notary Project` and keep pointing it to the website.

Reply:
One concern I've vaguely had but have had a hard time coming up with a productive solution for is... how do we distinguish this from a plugin-that-runs-in-Hashicorp-Vault-and-lets-Vault-users-sign-stuff-via-Notary from what it actually is, a plugin-that-runs-adjacent-to-Notary-and-lets-Notary-users-sign-stuff-with-keys-in-Vault?

We (Hashicorp) sadly tend to use the [Implicit: Hashicorp] "Vault Plugin: XYZ" ([eg 1](https://github.com/hashicorp/vault-plugin-secrets-kv/#vault-plugin-key-value-secrets-backend-) / [eg 2](https://github.com/hashicorp/vault-plugin-secrets-azure/#vault-plugin-azure-secrets-backend-)) as a way of saying the former, but this appears to be the pattern for the latter in this project... :laughing: 

Maybe "For use with Notation?" 

Sorry to cause problems? :-D

Reply:
> One concern I've vaguely had but have had a hard time coming up with a productive solution for is... how do we distinguish this from a plugin-that-runs-in-Hashicorp-Vault-and-lets-Vault-users-sign-stuff-via-Notary from what it actually is, a plugin-that-runs-adjacent-to-Notary-and-lets-Notary-users-sign-stuff-with-keys-in-Vault?
> 
> We (Hashicorp) sadly tend to use the [Implicit: Hashicorp] "Vault Plugin: XYZ" ([eg 1](https://github.com/hashicorp/vault-plugin-secrets-kv/#vault-plugin-key-value-secrets-backend-) / [eg 2](https://github.com/hashicorp/vault-plugin-secrets-azure/#vault-plugin-azure-secrets-backend-)) as a way of saying the former, but this appears to be the pattern for the latter in this project... 😆
> 
> Maybe "For use with Notation?"
> 
> Sorry to cause problems? :-D

Since this repo is under `notaryproject/notation-hashicorp-vault` and documents the usage in the README, I think it's clear enough for now?  (we mentioned `for Notation` in the doc and title as well.)